.github/workflows/test-pull-request.yml

```diff
@@ -16,45 +19,78 @@ on:
   pull_request:
     branches: [main]
 
+###############
+# Set the Job #
+###############
 jobs:
   validate:
+    name: Lint Code Base
```

Comment:
Need to remove line 27. It makes it look like the entire validation run is only for linting when you look at the checks in the PR and the actions log.

![Screen Shot 2022-07-21 at 2 54 48 PM](https://user-images.githubusercontent.com/13662379/180303869-ca1e9b90-9c8e-40ab-a21a-964659fa2221.png)

![Screen Shot 2022-07-21 at 2 53 52 PM](https://user-images.githubusercontent.com/13662379/180303662-68f86830-e902-49a7-8c4a-1c18da9abc75.png)

Reply:
Excellent! Got it. I think.